scipy/linalg/src/_batched_linalg.h

```diff
@@ -0,0 +1,305 @@
+#pragma once
+
+#include "Python.h"
+#include <limits>
+#include <cassert>
+#include "numpy/arrayobject.h"
+#include "npy_cblas.h"
+#include "_lapack_trampolines.h"
+#include "_npymath.h"
+
+
+using namespace _numpymath;
+
+
+// parroted from sqrtm
+// XXX can probably be replaced by ?copy from BLAS
```

Comment:
No unfortunately, this is basically to transpose a matrix in smaller chunks that fits into L1 cache to reduce the cache misses. 

At least in theory. I think we should unroll these to 8 for reals and 4 for complexes.I need a proper godbolt session to get a feeling for what compilers are choosing to do. I'll get to it eventually. 

 `dcopy` is BLAS' `memcpy`. They do different things.